README.md

```diff
@@ -14,6 +14,20 @@ Although this project is very new, it has been thoroughly tested in all modern b
 
 __[Demo](http://jakesidsmith.github.io/react-reorderable/)__
 
+## Installation
+
+1. Using npm
+  ```
+  npm install react-reorder
+  ```
+    Add `--save` or `-S` to update your package.json
+
+1. Using bower
+  ```
+  bower install react-reorderable
```

Comment:
I published to bower a while ago, and managed to get `react-reorderable`, but I don't think I should update now as people may already be using it.